util/types/types.go

```diff
@@ -0,0 +1,48 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package types
+
+var (
+	// HealthyRange represents the default flakiness thresholds for the
+	// flakiness report, and their corresponding colors (for displaying
+	// their table colors or something else)
+	HealthyRange = map[int]string{
+		20: "red",
+		10: "purple",
+		3:  "orange",
+		0:  "green",
```

Comment:
we actually have improved this in the original file, try to make consts instead of magic numbers here, aka,

```
var (
  highlyFlaky = 20
  somewhatFlaky = 10
  slightlyFlaky = 3
  nonFlaky = 0
)
```

and I don't think you will ever need the render colors here.

Reply:
Done